repositories/rmw_cyclonedds.BUILD.bazel

```diff
@@ -13,7 +13,10 @@ ros2_cpp_binary(
     ]),
     copts = ["-fvisibility=hidden"],
     includes = ["rmw_cyclonedds_cpp/include"],
-    linkopts = ["-fvisibility=hidden"],
+    linkopts = [
+        "-fvisibility=hidden",
+        "-latomic",
```

Comment:
-latomic should be defined as a linkopt of iceoryx. For the reference, this is a system library.